MemoryManagement/src/memorymanagementAlgorithm/Process.java

```diff
@@ -145,14 +197,19 @@ public void remove_Segment_i(int i) {
     public void clear_segment_vector() {
         getSegment_vector().clear();
     }
-
     //============print section=============================//
     public void print() {
         for (int i = 0; i < getSegment_vector().size(); i++) {
             getSegment_vector().get(i).print();
         }
     }
 
+    public void print(int i) {
+
+        getSegment_vector().get(i).print();

```

Comment:
Don't use functions to get class parameters in the class. to minimize stack overhead.

Reply:
need to print have another solution say it